code/modules/mob/living/carbon/human/login.dm

```diff
@@ -3,6 +3,8 @@
 	update_hud()
 	if(species)
 		species.handle_login_special(src)
+	if(client)
+		overlays -= image('icons/effects/effects.dmi', icon_state = "zzz_glow")
```

Comment:
Why isn't this using remove/add_overlay?

Reply:
Woops. Direct port fron Paradise. I'll fix it.